src/components/tree-view/tree-view.st.css

```diff
@@ -1,26 +1,51 @@
+:import{
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named: color_treeFocused, color_treeSelected;
+}
+
 .root {
-    font-family: 'Helvetica', Arial, sans-serif;
-    font-size: 18px;
+    font-family: Arial, sans-serif;
+    font-size: 16px;
     line-height: 1.5;
     text-align: left;
+    color: #000000;
     outline: 0 solid transparent;
 }
 
 .tree-node {
     -st-states: selected, focused;
     cursor: pointer;
-    display: inline;
-    padding: 2px 5px;
+    display: inline-block;
+    padding: 3px 7px 5px;
+    width: 228px;
```

Comment:
I would consider that as a reason to open discussion. 
Usually I expect tree nodes to be as long as my text is there.